setup.py

```diff
@@ -5,11 +5,6 @@
 import sys
 
 
-if sys.version_info < (3, 6):
-    print('adaptive requires Python 3.6 or above.')
-    sys.exit(1)
-
-
```

Comment:
Both are not incompatible. This check is still useful if you have users with `pip < 9.0.1` (which is still often the case for Python 2 users).